.env.example

```diff
@@ -19,26 +47,65 @@ CHRONOS_SCHEDULE_JOBS='0 */10 * * * *'
 CHRONOS_UPDATE_TOKENS='0 0 0 1,15 * *'
 CHRONOS_PROCESS_RECURRENCE='0 */5 * * * *'
 CHRONOS_AUTOPAUSE='0 0 5 * * *'
-CI='true'
+
+# DATABASES
+POSTGRES_PASSWORD=parabol
+POSTGRES_USER=pgparaboladmin
+POSTGRES_DB=parabol-saas
+POSTGRES_HOST=localhost
+POSTGRES_PORT=5432
+# POSTGRES_POOL_SIZE=5
+# POSTGRES_SSL_REJECT_UNAUTHORIZED=false
+# POSTGRES_SSL_DIR='/var/lib/postgresql'
+REDIS_PASSWORD=''
+REDIS_URL='redis://localhost:6379'
+# REDIS_TLS_CERT_FILE=./certs/redis.crt
+# REDIS_TLS_KEY_FILE=./certs/redis.key
+# REDIS_TLS_CA_FILE=./certs/redisCA.crt
+# REDIS_TLS_REJECT_UNAUTHORIZED='false'
+RETHINKDB_URL='rethinkdb://localhost:28015/actionDevelopment'
+RETHINKDB_SSL='false'
+
+# DATADOG
 DD_AGENT_HOST='172.17.0.1'
-DD_API_KEY='DD_API_KEY'
 DD_APM_ENABLED='false'
+DD_API_KEY=''
+DD_APPLICATIONID=''
+DD_APPSEC_ENABLED='true'
+DD_CLIENTTOKEN=''
+DD_ENV=''
+DD_LOGS_INJECTION='true'
+DD_RUNTIME_METRICS_ENABLED='true'
+DD_SERVICE='DD_SERVICE_NAME'
+DD_TRACE_AGENT_URL='PROTOCOL://URL:PORT'
```

Comment:
-1 it's confusing to have both this &  DD_AGENT_HOST since they are almost the same thing, but this one takes precedence. if this is the one we want to use then we should remove DD_AGENT_HOST and replace the value for this one to be the default value `http://localhost:8126`, or comment it out

Reply:
Removed DD_AGENT_HOST, set the DD_TRACE_AGENT_URL to the default and adding a link to the official documentation of dd-trace